From 68206f152d645da43178af453a3c1961d4aa0e06 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:04:51 -0400
Subject: [redhat] kabi: Adding symbol hrtimer_init (kernel/hrtimer.c)

Message-id: <c61c5d89e6a49ab6f5fb74a27e299c4d5ce894db.1412771859.git.poros@redhat.com>
Patchwork-id: 97184
O-Subject: [RHEL7.1 PATCH 31/67] kabi: Adding symbol hrtimer_init (kernel/hrtimer.c)
Bugzilla: 1148054
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1148054
    https://bugzilla.redhat.com/show_bug.cgi?id=1148054

    Description:
    ============
    Initialize hi res timers for sgi modules.
    Symbol used in extint_timer.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/hrtimer_init
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/hrtimer_init
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/hrtimer_init

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_init b/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_init
new file mode 100644
index 0000000..52ba60b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_init
@@ -0,0 +1,2 @@
+#1-
+0x09376b10 hrtimer_init vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/hrtimer_init b/redhat/kabi/kabi-module/kabi_s390x/hrtimer_init
new file mode 100644
index 0000000..99d51c1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/hrtimer_init
@@ -0,0 +1,2 @@
+#1-
+0x4c3c1b66 hrtimer_init vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_init b/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_init
new file mode 100644
index 0000000..e549547
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_init
@@ -0,0 +1,2 @@
+#1-
+0x25a97010 hrtimer_init vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1